src/Lime.Messaging/Contents/Invoice.cs

```diff
@@ -12,24 +13,35 @@ namespace Lime.Messaging.Contents
     public class Invoice : Document
     {
         public const string MIME_TYPE = "application/vnd.lime.invoice+json";
+        public const string TITLE_KEY = "title";
         public const string CREATED_KEY = "created";
         public const string METHODS_KEY = "methods";
         public const string DUE_TO_KEY = "dueTo";
         public const string CURRENCY_KEY = "currency";
         public const string TAXES_KEY = "taxes";
-        public const string TOTAL_KEY = "total";        
+        public const string TOTAL_KEY = "total";
         public const string ITEMS_KEY = "items";
+        public const string BILLED_TO_KEY = "billedTo";
```

Comment:
Use `billTo` instead.

```suggestion
        public const string BILL_TO_KEY = "billTo";
```